modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -250,85 +318,6 @@ func (suite *KeeperTestSuite) TestChanUpgradeTry() {
 	}
 }
 
-func (suite *KeeperTestSuite) TestChanUpgradeTry_CrossingHellos() {
```

Comment:
moved test cases above since it's the same test as above except crossing hello test cases call `path.EndpointB.ChanUpgradeInit()`